docs/ios-client/onboarding.md

```diff
@@ -9,8 +9,8 @@ As of Soundscape version 5.3.1 (October 2022):
 * macOS 12.6.1
 * Xcode 13.4.1
 * iOS 14.1
-* CocoaPods 1.11.3
-* CocoaPods Patch 1.0.2
+* CocoaPods 1.11.3 (since removed)
+* CocoaPods Patch 1.0.2 (since removed)
```

Comment:
The full text of this looks like this:
```md
## Supported Tooling Versions

As of Soundscape version 5.3.1 (October 2022):

* macOS 12.6.1
* Xcode 13.4.1
* iOS 14.1
* CocoaPods 1.11.3 (since removed)
* CocoaPods Patch 1.0.2 (since removed)
```
If stuff is going to get removed, I think the date and all the info should be updated too.
Is there some name for the current version of Soundscape? Additionally, does anybody know what the minimum supported versions of macOS, Xcode, and iOS are currently?

Reply:
Ah, I missed that context. I guess we should update the heading to "Soundscape Community version 1.0.1 (October 2023)" that will correspond to a future release that includes these changes.

I'm not sure if any changes have been made that would break compatibility with the specified macOS/iOS versions, but I suppose we could use the automated testing pipeline to confirm a minimum supported build environment.